radsat-sk/operation/subsystems/RDosimeter.c

```diff
@@ -203,9 +204,9 @@ int16_t dosimeterTemperature(dosimeterBoard_t board) {
 						DOSIMETER_RESPONSE_LENGTH, &dosimeterCommandBytes[adcChannel],
 						dataResponse, DOSIMETER_I2C_DELAY);
 

```

Comment:
fix indentation

Reply:
This style is correct.
The indentation here is meant illustrate that the indented code is part of the arguments of line 203 but needs to be on multiple lines because lines 203-205 combine to exceed 100 characters.